gallery/src/pages/components/ha-alert.markdown

```diff
@@ -13,16 +13,16 @@ The alert offers four severity levels that set a distinctive icon and color.
 
 <ha-alert alert-type="success">This is an success alert — check it out!</ha-alert>
 
-**Note:** This component is by [MUI](https://mui.com/components/alert/) and is not documented in the [Material Design guidelines](https://material.io).
+**Note:** This component is by <a href="https://mui.com/components/alert/" rel="noopener noreferrer target="_blank">MUI</a> and is not documented in the <a href="https://material.io" rel="noopener noreferrer target="_blank">Material Design guidelines</a>.
```

Comment:
Anchors do not seem to work

Reply:
I changed this to open links in a new tab